x11-libs/libxaw/libxaw-1.0.13.recipe

```diff
@@ -0,0 +1,117 @@
+SUMMARY="X Athena Widget library"
+DESCRIPTION="X Athena Widgets, or Xaw, is a widget set based on the X Toolkit \
+Intrinsics (Xt) Library. Xaw & Xt form together what is known as the X Toolkit."
+HOMEPAGE="https://www.x.org/releases/individual/lib/"
+COPYRIGHT="1985-1990, 1994, 1998 The Open Group
+	2003-2004 Roland Mainz
+	1998-1999 The XFree86 Project, Inc.
+	1991 OMRON Corporation
+	1985-1998 Digital Equipment Corporation
+	1989 Prentice Hall
+	1985-1989, 1991, 1994 X Consortium"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXaw-$portVersion.tar.bz2"
+CHECKSUM_SHA256="8ef8067312571292ccc2bbe94c41109dcf022ea5a4ec71656a83d8cce9edb0cd"
+SOURCE_DIR="libXaw-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libXaw6Version="6.0.1"
+libXaw7Version="7.0.0"
+
+libXaw6VersionCompat="$libXaw6Version compat >= ${libXaw6Version%%.*}"
+libXaw7VersionCompat="$libXaw7Version compat >= ${libXaw7Version%%.*}"
```

Comment:
Hmm, that's a good idea. But I would prefer a more symetric variant:
```
libVersion6="6.0.1"
libVersion7="7.0.0"
```
(BTW, thanks @alaviss for your review! I'll answer your other comments asap.)

Reply:
Thanks @alaviss for your review!

You gave me a good idea. We can have a shorter name for the variables.
But since `libXaw.so.8` will probably never be used, I think we can stick to a hardcoded "6" and  "7". The recipe will probably remain valid forever with these hardcoded major versions.
(I'll update the PR asap.)